intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
```

Comment:
Right now they have to click `Yes`. I think you're asking, can we remove that step.
If it's possible, +1 to that idea...

Until then, see the latest commit.